code/EnvironmentChecker.php

```diff
@@ -88,11 +88,18 @@ function canAccess($member = null, $permission = "ADMIN") {
 	function index() {
 		$response = new SS_HTTPResponse;
 		$result = EnvironmentCheckSuite::inst($this->checkSuiteName)->run();
-		
+
+		// output the result as JSON if requested
+		if($this->getRequest()->getVar('format') == 'json') {
```

Comment:
would it be better to check for the standard http header `Content-Type: application/json;` ?

A lot of REST interfaces works like that.